kda/graph_utils.py

```diff
@@ -196,7 +196,7 @@ def _is_ccw(cycle, start, end):
     for i in range(len(double) - 1):
         if (double[i], double[i + 1]) == (start, end):
             return True
-    return None
+    return False
```

Comment:
I don't know why this was set to return `None` before -- this should only ever return `True`/`False`.